uxarray/grid.py

```diff
@@ -38,8 +37,8 @@ def __init__(self, *args, **kwargs):
         Parameters
         ----------
 
-        data_arg : string, ndarray, list, tuple, required
-            - Input file name with extension or
+        data_arg : xarray.Dataset, ndarray, list, tuple, required
+            - Input xarray.Dataset or
```

Comment:
```suggestion
        dataset : xarray.Dataset, ndarray, list, tuple, required
            - Input xarray.Dataset or
```

really minor comment...